README.md

```diff
@@ -100,8 +100,8 @@ module.exports = {
       '/absolute/path/to/dir',
       '**/*',
       {
-        from: '**/*',
-        globOptions: {
```

Comment:
dont remove `globOptions`, this plugin supports this options 
https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/preProcessPattern.js#L36